tests/queries/0_stateless/03705_function_dict_get_keys_multiple_dict_and_no_caching.sql

```diff
@@ -0,0 +1,102 @@
+CREATE TABLE src_products
```

Comment:
Nit: consider dropping tables and dictionaries to avoid errors after a repetitive run in manual tests.